Modules/rat_rescue.py

```diff
@@ -108,7 +121,12 @@ def __init__(self, uuid: UUID,
         self._board_index = board_index
         self._lang_id = lang_id
         self._status = status
-        self._hash = None
+        self._mecha_uuid = mecha_id
```

Comment:
This is a nitpick, but could you decide on either `mecha_id` or `mecha_uuid`?
Furthermore, I think a it should have a getter so it can be used in more places.

Reply:
*blinks* yeah that was supposed to be `mecha_uuid`. will fix